xarray/plot/dataarray_plot.py

```diff
@@ -984,29 +992,29 @@ def newplotfunc(
                 ckw = {vv: cmap_params[vv] for vv in ("vmin", "vmax", "norm", "cmap")}
                 cmap_params_subset.update(**ckw)
 
-        if z is not None:
-            if ax is None:
-                subplot_kws.update(projection="3d")
-            ax = get_axis(figsize, size, aspect, ax, **subplot_kws)
-            # Using 30, 30 minimizes rotation of the plot. Making it easier to
-            # build on your intuition from 2D plots:
-            plt = import_matplotlib_pyplot()
-            if Version(plt.matplotlib.__version__) < Version("3.5.0"):
-                ax.view_init(azim=30, elev=30)
+        with plt.rc_context(_styles):
```

Comment:
Doesn't this override any user style settings in `.matplotlibrc`?

Reply:
Yes, the innermost context takes priority. `edgecolor="k"` can be used though. 
We have the same problem with color maps:

```python
ds = xr.tutorial.scatter_example_dataset(seed=42)

fig, axs = plt.subplots(1, 2)
with plt.rc_context(
    {
        "scatter.edgecolors": "k",
        "image.cmap": "Greys",
    }
):
    # Normal scatter:
    axs[0].scatter(ds.A.values.ravel(), ds.B.values.ravel(), c=3 * ds.B.values)

    # xarray scatter:
    ds.plot.scatter(x="A", y="B", ax=axs[1])
```
![image](https://user-images.githubusercontent.com/14371165/208303720-223080a4-e8b4-4851-985d-28d0ee2bd178.png)